lib/config/presets/internal/replacements.ts

```diff
@@ -51,6 +52,18 @@ export const presets: Record<string, Preset> = {
     ],
     ignoreDeps: [], // Hack to improve onboarding PR description
   },
+  'airbnb-prop-types-to-prop-types-tools': {
```

Comment:
```suggestion
  'airbnb-prop-types-to-prop-types-tools': { // eslint-disable-line sort-keys
```

I hope the above correctly disables the `sort-keys` rule for just the `'airbnb-prop-types-to-prop-types-tools'` object.

If the above does not work, maybe we can try this format with the _next line_ variant:

```ts
// eslint-disable-next-line sort-keys
  'airbnb-prop-types-to-prop-types-tools': { 
```

Read the [ESLint docs, disabling rules](https://eslint.org/docs/latest/use/configure/rules#disabling-rules) to learn more.